library-benchmarks/build.gradle

```diff
@@ -27,12 +27,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'realm-android'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 27
+    buildToolsVersion "27.0.0"
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 22 // Below 23 to avoid new permission system introduced in M
+        targetSdkVersion 27
```

Comment:
No need to add a code for runtime permission?

Reply:
Tried, it doesn't ask for runtime permissions. Not sure why the comment was there.